### PR TITLE
feat: add requirePlugin.async

### PR DESCRIPTION
**Pull request by bytemain**

### CODE REVIEW

File: packages/global/types/lib.global.d.ts [hunk 1]
```diff
@@ -21,6 +29,13 @@ export type IMixin4Legacy<
   ExtraOptions
 >;
 
+export interface Require {
+  <T extends any>(path: string): T;
+  <T extends any>(path: string, cb?: (o: T) => void): void;
+
+  async<T extends any>(id: string): Promise<T>;
```
Comment (GodD6366) [inline packages/global/types/lib.global.d.ts:36]: 统一下吧 id -> path

Comment (bytemain) [reply packages/global/types/lib.global.d.ts]: done

**Review by GodD6366 — APPROVED** [review]
LGTM

**Review by GodD6366 — APPROVED** [review]
LGTM

**Pull request merged**